### DISCUSSION TITLE
Dbeaver 21.3.3 giving issue while connecting the postgresql DB

### DISCUSSION THREAD

**Discussion by rupesh81ad** [dbeaver/dbeaver#19353] [Q&A] [answered]
### Description

Hi,
While connecting the postgresql DB , it is giving the driver issue to connect
![dbeaver_issue](https://user-images.githubusercontent.com/125982153/220353519-42ca4f25-a34c-4a19-aba8-9e1bf28ca0ab.JPG)

### DBeaver Version

21.3.3

### Operating System

Window 10 Enterprise

**Comment by E1izabeth** [comment]
Could you send me the screenshot of your driver libraries?
![image](https://user-images.githubusercontent.com/28875055/220356265-57f0d1fd-e39e-4919-ac2e-0b5c11a66880.png)

**Comment by rupesh81ad** [comment]
Hi,
please can you share the step to get this windows, i am not able get this window.

**Comment by rupesh81ad** [comment]
![dbeaver_issue_1](https://user-images.githubusercontent.com/125982153/220358142-744d5073-2834-4c09-8e7c-0286fdc8970a.JPG)

**Comment by rupesh81ad** [comment]
![dbeaver_issue_2](https://user-images.githubusercontent.com/125982153/220358749-a829738f-8ab9-403d-b784-0c824f1e6f79.JPG)

**Comment by rupesh81ad** [comment]
Hi,
Please check now

**Comment by E1izabeth** [comment]
Thank you. It seems that the drivers wasn't downloaded. You can select all blue items and press Download/Update button. Then after successful downloading it should work.

**Comment by rupesh81ad** [comment]
![dbeaver_issue_3](https://user-images.githubusercontent.com/125982153/220362133-9919ec53-27c6-4356-82ee-68e90967996c.JPG)

**Comment by rupesh81ad** [comment]
Hi,
After selecting the file an pressing the download and update button , giving this error message and cannot do it.

**Comment by E1izabeth** [comment]
Please check this setting
![image](https://user-images.githubusercontent.com/28875055/220362850-c88ac84a-2b7e-499d-b9b9-0e9efc6e871f.png)
If there is http://repo1.maven.org/ then remove it and add https://repo1.maven.org/maven2/ instead

**Comment by rupesh81ad** [comment]
Already this entry is present

**Comment by rupesh81ad** [comment]
![dbeaver_issue_4](https://user-images.githubusercontent.com/125982153/220365491-5a4a8fd4-77d8-4f19-adc6-ddefa15e1daf.JPG)

**Comment by E1izabeth** [comment]
Hmm
Probably trying the latest DBeaver version (22.3.5) might help.

**Comment by rupesh81ad** [comment]
Hi,
Please can you share the link for free download.

**Comment by rupesh81ad** [comment]
Just to update for 22.3.5 is also same issue

**Comment by E1izabeth** [comment]
Probably downloading and adding driver from here https://jdbc.postgresql.org/download/ would be enough.
After downloading file from this website, you need to remove all blue items and add the new one.

**Comment by rupesh81ad** [comment]
Please can you assist, we will have call and do setup, it is very much needed for our project

**Comment by E1izabeth** [comment, marked as answer]
1. Press download button here 
![image](https://user-images.githubusercontent.com/28875055/220394011-93aee79a-4ad4-451b-9149-dc4e7a01b03f.png)

2. Delete proposed libraries which you can't download for some reason by selecting them and pressing Delete button
![image](https://user-images.githubusercontent.com/28875055/220394227-184476f0-fa04-4854-b874-58fb41f71880.png)

3. Add downloaded file with Add File button
![image](https://user-images.githubusercontent.com/28875055/220394459-144b5087-12c2-4a74-a934-223d08c88628.png)

4. Press OK and try to connect to your database

**Comment by rupesh81ad** [comment]
Yes it worked. Thanks very much for your support.
Really appreciate your good response.

**Comment by E1izabeth** [comment]
@rupesh81ad could you test if you can open this url https://repo1.maven.org/maven2/ in your web browser? I'd like to be sure about the reason that you can't download the drivers from maven.
